listenbrainz/webserver/views/api.py

```diff
@@ -426,6 +431,110 @@ def delete_listen():
     return jsonify({'status': 'ok'})
 
 
+def serialize_playlists(playlists, playlist_count, count, offset):
+    """
+        Serialize the playlist metadata for the get playlists commands.
+    """
+
+    items = []
+    for playlist in playlists:
+        items.append(serialize_jspf(playlist))
+
+    return {"playlists": items,
+            "playlist_count": playlist_count,
+            "offset": offset,
+            "count": count}
+
+
+@api_bp.route("/user/<playlist_user_name>/playlists", methods=["GET", "OPTIONS"])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+def get_playlists_for_user(playlist_user_name):
+    """
+    Fetch the playlists for a user. Returns an array of playlists without their recordings. If
+    a user token is provided in the Authorization header, return private playlists as well
+    as public playlists for that user.
+
+    :params count: The number of playlists to return (for pagination). Default
+        :data:`~webserver.views.api.DEFAULT_NUMBER_OF_PLAYLISTS_PER_CALL`
+    :params offset: The offset of into the list of playlists to return (for pagination)
+    :statuscode 200: Yay, you have data!
+    :statuscode 404: User not found
+    :resheader Content-Type: *application/json*
+    """
+    user = validate_auth_header(True)
+
+    count = request.args.get('count', DEFAULT_NUMBER_OF_PLAYLISTS_PER_CALL)
+    offset = request.args.get('offset', 0)
+    playlist_user = db_user.get_by_mb_id(playlist_user_name)
+    if playlist_user is None:
+        raise APINotFound("Cannot find user: %s" % playlist_user_name)
+
+    include_private = True if user and user["id"] == playlist_user["id"] else False
+    playlists, playlist_count = db_playlist.get_playlists_for_user(playlist_user["id"], include_private, False, count, offset)
```

Comment:
I'd like to see named parameters here to know that the `False` refers to load_recordings without having to look at that function definition